docs/source/eFeatures.rst

```diff
@@ -88,7 +88,7 @@ LibV5 : all_ISI_values, inv_first_ISI, inv_second_ISI, inv_third_ISI, inv_fourth
 1.0 over first/second/third/fourth/fith/last ISI; returns 0 when no ISI
 
 - **Required features**: peak_time (ms)
-- **Units**: Hz
+- **Units**: ms
```

Comment:
So the problem here is that all_ISI_values is in ms, while the other inv_*_ISI are in Hz. Maybe leave Hz here and create a new entry for all_ISI_values?